src/models/log/reducer.js

```diff
@@ -1,63 +1,131 @@
 import {
-  check,
-  fold,
-  raise,
-  replace,
-  startGame,
-  nextPhase,
+  gameWin,
+  gameLose,
+  gameTie,
+  gameChecked,
+  cardReplaced,
+  gameStarted,
+  gameFolded,
+  betRaised,
+  phaseAdvanced,
+  betNotRaised,
 } from "models/game/actions";
 
+import { handCheckToMsg } from "lib/handCheck";
+
 const initialState = {
   logger: [],
 };
 
 const logReducer = (state = initialState, action) => {
   switch (action.type) {
-    case startGame.type: {
+    case gameStarted.type: {
       const newLogger = state.logger.slice();
       newLogger.push("Game started.");
-
+      newLogger.push("You have " + action.payload.playerMoney + " euros.");
+      newLogger.push("Your bet is " + action.payload.playerBet + " euros.");
+      newLogger.push(
+        "You have " + handCheckToMsg(action.payload.playerHand) + "."
+      );
       return {
         logger: newLogger,
       };
     }
 
-    case fold.type: {
+    case gameFolded.type: {
       const newLogger = state.logger.slice();
       newLogger.push("Player folds.");
-
+      newLogger.push("You have " + action.payload.playerMoney + " euros.");
       return {
         logger: newLogger,
       };
     }
 
-    case raise.type: {
+    case betRaised.type: {
       const newLogger = state.logger.slice();
       newLogger.push("Player raises.");
-
+      newLogger.push("You have " + action.payload.playerMoney + " euros.");
+      newLogger.push("Your bet is " + action.payload.playerBet + " euros.");
+      return {
+        logger: newLogger,
+      };
+    }
+    case betNotRaised.type: {
+      const newLogger = state.logger.slice();
+      newLogger.push("You cannot bet anymore.");
       return {
         logger: newLogger,
       };
     }
 
-    case replace.type: {
+    case cardReplaced.type: {
       const newLogger = state.logger.slice();
       newLogger.push("Player trades a card.");
-
+      newLogger.push(
+        "You now have " + handCheckToMsg(action.payload.playerHand) + "."
+      );
```

Comment:
Try instead of slicing to use copy and array spread methods rg const newArray = [...previousArray, 'Another Message'].
Even better, create an action that receives a single string or array of strings and an epic should be responsible to append the logger messages

Reply:
It has been done.